src/ontology/fbbt-edit.obo

```diff
@@ -118272,6 +118272,41 @@ intersection_of: part_of FBbt:00058008 ! larval anterior-middle midgut junction
 created_by: http://orcid.org/0000-0002-6095-8718
 creation_date: 2021-04-14T13:26:51Z
 
+[Term]
+id: FBbt:00058013
+name: TE neuron
```

Comment:
Use long-form as label i.e. 'transient extrinsic neuron' and have this as the synonym. Might be even better to call it 'transient extrinsic neuron of pupal optic lobe' - this gives curators more information from the label.